docs/source/api/inference.rst

```diff
@@ -65,3 +72,35 @@ ADVI minibatch
 
 .. automodule:: pymc3.variational.advi_minibatch
    :members:
+
+Inference
+^^^^^^^^^
+
+.. currentmodule:: pymc3.variational.operators
+
+.. automodule:: pymc3.variational.operators
+   :members:
+
+OPVI
+^^^^
+
+.. currentmodule:: pymc3.variational.opvi
+
+.. automodule:: pymc3.variational.opvi
+   :members:
+
+SVGD
+^^^^
```

Comment:
I think @ferrine has re-implemented these in OPVI, so we can probably drop SVGD for now.